pkgs/top-level/all-packages.nix

```diff
@@ -25336,6 +25338,8 @@ with pkgs;
     harfbuzz = harfbuzzFull;
   };
 
+  tectonic-with-biber = callPackage ../tools/typesetting/tectonic/wrapper.nix { };
```

Comment:
I still tend towards editing the regular `tectonic` expression, such that `tectonic.unwrapped` is the unwrapped tectonic, and `tectonic` is wrapped with `biber-for-tectonic`, using `symlinkJoin`...

Reply:
I am not sure about this, because biber is popular but not _that_ popular (unfortunately most journals still use bibtex). So it would be simpler to maintain a different package, for people that do need biber. But if you really think we should make the biber-wrapped tectonic as the default, I am also okay with that!